explainaboard/metrics/metric.py

```diff
@@ -371,16 +371,39 @@ def get_batch_data(self) -> np.ndarray[tuple[int, int, int], Any]:
 
 
 class Metric(metaclass=abc.ABCMeta):
-    """A class representing an evaluation metric."""
+    """A class representing an evaluation metric.
+
+    When a subclass needs to construct a random number generator, initialize the random
+    generator as follows: Invoke the `get_seed()` method to get a numpy SeedSequence,
+    spawn the SeedSequence with the SeedSequence's `spawn` method, and then pass the
+    spawned SeedSequence to the random generator's constructor.
```

Comment:
Better to organize this section using enumeration.

Reply:
Done.